packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -0,0 +1,304 @@
+
+    const landingCompanyName = `(${
+        companyNamesAndUrls?.[selectedJurisdiction as keyof typeof companyNamesAndUrls]?.shortcode
+    })`;
+
+    const onSubmit = useCallback(async () => {
+        const accountType = marketType === 'synthetic' ? 'gaming' : marketType;
+
+        // in order to create account, we need to set a password through trading_platform_password_change endpoint first
+        // then only mt5_create_account can be called, otherwise it will response an error for password required
+        if (isMT5PasswordNotSet) {
+            await tradingPasswordChange({
+                new_password: password,
+                platform: PlatformDetails.mt5.platform,
+            });
+        }
+
+        const categoryAccountType = activeTrading?.is_virtual ? 'demo' : accountType;
+
+        mutate({
+            payload: {
+                account_type: categoryAccountType,
+                address: settings?.address_line_1 ?? '',
+                city: settings?.address_city ?? '',
+                company: selectedJurisdiction,
+                country: settings?.country_code ?? '',
+                email: settings?.email ?? '',
+                leverage: availableMT5Accounts?.find(acc => acc.market_type === marketType)?.leverage ?? 500,
+                mainPassword: password,
+                ...(marketType === 'financial' && { mt5_account_type: 'financial' }),
+                ...(selectedJurisdiction &&
+                    (selectedJurisdiction !== 'labuan'
+                        ? {
+                              account_type: categoryAccountType,
+                              ...(selectedJurisdiction === 'financial' && { mt5_account_type: 'financial' }),
+                          }
+                        : {
+                              account_type: 'financial',
+                              mt5_account_type: 'financial_stp',
+                          })),
+                ...(marketType === 'all' && { sub_account_category: 'swap_free' }),
+                name: settings?.first_name ?? '',
+                phone: settings?.phone ?? '',
+                state: settings?.address_state ?? '',
+                zipCode: settings?.address_postcode ?? '',
+            },
+        });
+    }, [
+        activeTrading?.is_virtual,
+        availableMT5Accounts,
+        isMT5PasswordNotSet,
+        marketType,
+        mutate,
+        password,
+        selectedJurisdiction,
+        settings?.address_city,
+        settings?.address_line_1,
+        settings?.address_postcode,
+        settings?.address_state,
+        settings?.country_code,
+        settings?.email,
+        settings?.first_name,
+        settings?.phone,
+        tradingPasswordChange,
+    ]);
+
+    const renderTitle = useCallback(() => {
+        if (isSuccess) {
+            return ' ';
+        }
+        return `${hasMT5Account ? 'Add' : 'Create'} a ${isDemo ? 'demo' : 'real'} ${PlatformDetails.mt5.title} account`;
```

Comment:
make aan Enum in constant file here packages/tradershub/src/features/cfd/constants.tsx and name it AccountType to choose between `real and demo`

Reply:
`AccountType` or `Category` ...? iinm we have enums in cfd package for demo/real but it's called `Category`